src/main/java/org/prebid/server/floors/BasicPriceFloorResolver.java

```diff
@@ -189,21 +190,25 @@ private static PriceFloorModelGroup extractFloorModelGroup(PriceFloorRules floor
 
     private static <V> Map<String, V> keysToLowerCase(Map<String, V> map) {
         return map.entrySet().stream()
-                .collect(Collectors.toMap(entry -> entry.getKey().toLowerCase(), Map.Entry::getValue));
+                .collect(
+                        HashMap::new,
+                        (hashMap, entry) -> hashMap.put(entry.getKey().toLowerCase(), entry.getValue()),
+                        HashMap::putAll);
```

Comment:
Why do we need this change?

Reply:
so we should support null value for the price floor, and there is a bug https://bugs.openjdk.org/browse/JDK-8148463 that doesn't allow to put null value as a map value using Collectors.toMap()

this is a workaround

Reply:
Maybe this would be a little prettier:
```java
                .collect(Collectors.toMap(
                        entry -> entry.getKey().toLowerCase(),
                        Map.Entry::getValue,
                        (first, second) -> second,
                        HashMap::new));
```

Reply:
it leads to the same NPE issue I described above